### PR TITLE
Deeptmhmm

### PR DESCRIPTION
**Pull request by vagkaratzas**

### CODE REVIEW

File: modules/ebi-metagenomics/deeptmhmm/meta.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/yaml-schema.json
+name: deeptmhmm
+description: A Deep Learning Model for Transmembrane Topology Prediction and Classification
+keywords:
+  - sort
```
Comment (mberacochea) [inline modules/ebi-metagenomics/deeptmhmm/meta.yml:6]: sort, example? I think you forgot the keyworks

File: modules/ebi-metagenomics/deeptmhmm/meta.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  - fasta:
+      type: file
+      description: Database of sequences in FASTA format
+      pattern: "*.{fasta,fa,fasta.gz,fa.gz}"
+
+output:
+  - gff3:
+      type: file
+      description: Predicted topologies in general Feature Format Version 3
+      pattern: "biolib_results/TMRs.gff3"
```
Comment (mberacochea) [inline modules/ebi-metagenomics/deeptmhmm/meta.yml:27]: Is this correct? According to the docs it should be:

- input/output patterns (if present) MUST follow a [Java glob pattern](https://docs.oracle.com/javase/tutorial/essential/io/fileOps.html#glob)

Comment (mberacochea) [reply modules/ebi-metagenomics/deeptmhmm/meta.yml:27]: should only be for the file? and not the path?

**Review by mberacochea — CHANGES_REQUESTED** [review]
Great stuff... keep them coming. I've added some comments

File: modules/ebi-metagenomics/deeptmhmm/meta.yml (hunk 1 above)
Comment (vagkaratzas) [reply modules/ebi-metagenomics/deeptmhmm/meta.yml]: I blame the linting. Should have reminded me :P

File: modules/ebi-metagenomics/deeptmhmm/meta.yml (hunk 2 above)
Comment (vagkaratzas) [reply modules/ebi-metagenomics/deeptmhmm/meta.yml:27]: Nah this is more like it. I updated it with some more info and also added the optional outputs I had forgotten.

**Review by mberacochea — APPROVED** [review]
:shipit:

**Pull request merged**